src/System.Dynamic.Runtime/src/System.Dynamic.Runtime.csproj

```diff
@@ -7,9 +7,9 @@
     <ProjectGuid>{C4E89B8C-07DB-40CA-8C99-82A23E8F5F39}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.Dynamic.Runtime</AssemblyName>
-    <AssemblyVersion>4.0.10.0</AssemblyVersion>
-    <DefineConstants>$(DefineConstants);FEATURE_CORECLR</DefineConstants>
     <RootNamespace>System.Dynamic.Runtime</RootNamespace>
+    <AssemblyVersion>4.0.10.0</AssemblyVersion>
+    <DefineConstants Condition=" '$(IsInterpreting)' != 'true' " >$(DefineConstants);FEATURE_CORECLR</DefineConstants>
```

Comment:
I am considering that since what we have currently is somewhat confusing, but perhaps as a separate change. I do not want this change to get big since it is kind of tricky and cosmetic changes can wait.